### ISSUE TITLE
Drop dependency on starman

### ISSUE THREAD

**Issue by mtoma** [dotse/zonemaster-backend#397]
Starman appears not to be useful enough to be kept as dependency.
Dropping the dependency will simplify a lot the installation procedure.

**Comment by matsduf** [comment]
@mtoma, what issues do you see with Starman?

Do you have a suggestion for a replacement?

**Comment by mtoma** [comment]
Yes, my suggestion would be to drop it in favor of a standard webserver, either Apache or Nginx or whatever is considered to be standard in the target distribution. I don't see any reason why anyone would accept to install Starman and at the same time refuse to install Apache or Nginx.

**Comment by matsduf** [comment]
Isn't the current solution simpler to install than a solution based on a HTTP server?

What problems do you see with the Starman solution? What would we gain by replacing to with a HTTP server solution?

**Comment by mattias-p** [comment]
I just stumbled upon this while looking for something else: https://metacpan.org/pod/Plack::Handler::Apache2

**Comment by matsduf** [comment]
> I just stumbled upon this while looking for something else: https://metacpan.org/pod/Plack::Handler::Apache2

It is interesting to see that there are alternatives. The downside of using Apache HTTPD is that the installation of a stand-alone Backend will be more complex. If GUI is installed on the same server as Backend, then there is probably not much difference.
